prediction_market_agent_tooling/markets/omen/omen_contracts.py

```diff
@@ -108,6 +142,13 @@ def does_condition_exists(
     ) -> bool:
         return self.getOutcomeSlotCount(condition_id) > 0
 
+    def payoutDenominator(self, condition_id: HexAddress) -> int:
```

Comment:
`getConditionId` return `HexBytes`

```suggestion
    def payoutDenominator(self, condition_id: HexBytes) -> int:
```